src/garage/tf/algos/ddpg.py

```diff
@@ -271,7 +271,6 @@ def train_once(self, itr, paths):
         for _ in range(self.n_train_steps):
             if (self.replay_buffer.n_transitions_stored >=
                     self.min_buffer_size):
-                self.evaluate = True
```

Comment:
You might want to keep this flag here

Reply:
maybe you should change the name to avoid confusion, like `self.start_train` or sth like that